lib/plugins/entry-files-manifest.js

```diff
@@ -9,26 +9,42 @@
 
 'use strict';
 
-const EntryFilesManifestPlugin = require('../webpack/entry-files-manifest-plugin');
 const PluginPriorities = require('./plugin-priorities');
-const path = require('path');
 const sharedEntryTmpName = require('../utils/sharedEntryTmpName');
-const manifestKeyPrefixHelper = require('../utils/manifest-key-prefix-helper');
+const AssetsPlugin = require('assets-webpack-plugin');
+
+function cleanFilePaths(assets) {
```

Comment:
the function name looks weird, as it is not only cleaning paths, but also stringifying the output to JSON.

Reply:
Maybe `processOutput(assets)` instead?
It's a bit vague but it would match the name of the related option of the `assets-webpack-plugin`.